content/BatchResponseContentCollection.md

```diff
@@ -0,0 +1,24 @@
+# BatchResponseContentCollection
+
+## Objectives
+
+Provide component to simplify the processing of multiple batch responses made via `BatchRequestContentCollection`.
+
+## Requirements
+
+See [constraints](ContentArchitecturalConstraints.md) related to all content objects.
+
+- Hold multiple batch response objects (i.e. `BatchResponseContent`) made from a collection of batch request objects(i.e. `BatchRequestContentCollection`)
+- Allow users to enumerate collection of individual response objects
+- Allow users to retrieve a specific response object by using the same unique identifiers used in making the request.
```

Comment:
See comments earlier about uniqueness. We need to make sure uniqueness is at the collection level, not just at the batch content.

Reply:
Clarified this in the latest updates.